tailwind.config.ts

```diff
@@ -0,0 +1,29 @@
+import { join } from 'path'
+import type { Config } from 'tailwindcss'
+
+// 1. Import the Skeleton plugin
+import { skeleton } from '@skeletonlabs/tw-plugin'
+
+const config = {
+    // 2. Opt for dark mode to be handled via the class method
+    darkMode: 'class',
+    content: [
+        './src/**/*.{html,js,svelte,ts}',
+        // 3. Append the path to the Skeleton package
+        join(
+            require.resolve('@skeletonlabs/skeleton'),
+            '../**/*.{html,js,svelte,ts}'
+        ),
+    ],
+    theme: {
+        extend: {},
+    },
+    plugins: [
+        // 4. Append the Skeleton plugin (after other plugins)
+        skeleton({
+            themes: { preset: ['vintage'] },
```

Comment:
I think this line "installs" the theme, but then it gets applied at the top level of the app on the `<body>` element.